src/App.tsx

```diff
@@ -305,120 +311,126 @@ function App() {
   const hasActiveV1Bonds = accountBonds.length > 0;
 
   return (
-    <ThemeProvider theme={themeMode}>
-      <CssBaseline />
-      <div className={`app ${isSmallerScreen && "tablet"} ${isSmallScreen && "mobile"} ${theme}`}>
-        <Messages />
-        <TopBar theme={theme} toggleTheme={toggleTheme} handleDrawerToggle={handleDrawerToggle} />
-        <nav className={classes.drawer}>
-          {isSmallerScreen ? (
-            <NavDrawer mobileOpen={mobileOpen} handleDrawerToggle={handleDrawerToggle} />
-          ) : (
-            <Sidebar />
-          )}
-        </nav>
-
-        <div className={`${classes.content} ${isSmallerScreen && classes.contentShift}`}>
-          {oldAssetsDetected &&
-            !hasActiveV1Bonds &&
-            trimmedPath.indexOf("dashboard") === -1 &&
-            oldAssetsEnoughToMigrate && <CallToAction setMigrationModalOpen={setMigrationModalOpen} />}
-
-          <Switch>
-            <Route exact path="/dashboard">
-              <TreasuryDashboard />
-            </Route>
-
-            <Route exact path="/">
-              <Redirect to="/stake" />
-            </Route>
-
-            <Route path="/stake">
-              {/* if newAssets or 0 assets */}
-              {newAssetsDetected || (!newAssetsDetected && !oldAssetsDetected) || !oldAssetsEnoughToMigrate ? (
-                <Stake />
-              ) : (
+    <MultifarmProvider token={MULTIFARM_API_KEY} provider="olympus" theme="olympus" themeColors={theme}>
+      <ThemeProvider theme={themeMode}>
```

Comment:
Should multifarmprovider be nested underneath Theme provider?

Reply:
It's probably cleaner to add it under ThemeProvider (I will update). Just a note, I tried it both ways and it doesn't seem to matter either way, because the theme styles for multifarm live in the multifarm library, we just pass light or dark to the themeColors property.